podcasts/End of Year/Stories/ListeningTimeStory.swift

```diff
@@ -1,27 +1,111 @@
 import SwiftUI
+import PocketCastsServer
+import PocketCastsDataModel
 
 struct ListeningTimeStory: StoryView {
     var duration: TimeInterval = 5.seconds
 
     let listeningTime: Double
 
+    let podcasts: [Podcast]
+
     var body: some View {
-        ZStack {
-            Color.black
+        GeometryReader { geometry in
+            ZStack {
+                DynamicBackgroundView(podcast: podcasts[0])
+
+                VStack {
+                    Text(L10n.eoyStoryListenedTo(listeningTime.localizedTimeDescription ?? ""))
+                        .foregroundColor(.white)
+                        .font(.system(size: 25, weight: .heavy))
+                        .foregroundColor(.white)
+                        .multilineTextAlignment(.center)
+                        .frame(maxHeight: geometry.size.height * 0.12)
+                        .minimumScaleFactor(0.01)
+
+                    Text(FunnyTimeConverter.timeSecsToFunnyText(listeningTime))
+                        .font(.system(size: 15, weight: .bold))
+                        .foregroundColor(.white)
+                        .multilineTextAlignment(.center)
+                        .frame(maxHeight: geometry.size.height * 0.07)
+                        .minimumScaleFactor(0.01)
+                        .opacity(0.8)
+                    Spacer()
+                }
+                .padding(.top, geometry.size.height * 0.15)
+                .padding(.trailing, 40)
+                .padding(.leading, 40)
+
+                VStack {
+                    Spacer()
+
+                    HStack {
+                        ForEach([1, 0, 2], id: \.self) {
+                            podcastCover($0)
+                        }
+                    }
+                    .modifier(PodcastCoverPerspective())
+                    .position(x: geometry.frame(in: .local).midX, y: geometry.size.height - 230)
+                }
+            }
 
             VStack {
-                Text("In 2022, you spent \(listeningTime.localizedTimeDescription ?? "") listening to podcasts")
-                    .foregroundColor(.white)
-                Text(FunnyTimeConverter.timeSecsToFunnyText(listeningTime))
-                    .foregroundColor(.white)
+                Spacer()
+                HStack {
+                    Spacer()
+                    Image("logo_white")
+                        .padding(.bottom, 40)
+                    Spacer()
+                }
+            }
+        }
+    }
+
+    @ViewBuilder
+    func podcastCover(_ index: Int) -> some View {
+        Group {
+            if let podcast = podcasts[safe: index] {
+                ImageView(ServerHelper.imageUrl(podcastUuid: podcast.uuid, size: 280))
+            } else {
+                Rectangle().opacity(0)
             }
```

Comment:
> A simple idea could be to just show a rectangle with the podcast color while it's loading in.

I think we can add a black translucent square. The colors are tricky because the podcast might have them or not, and if they don't we need to make a request to retrieve them.